### ISSUE TITLE
Add ERC20 tokens to Ethereum Whitelist

### ISSUE THREAD

**Issue by pandaring2you** [Sifchain/sifnode#1843]
https://www.notion.so/sifchain/Whitelisting-ERC20-Token-Queue-22335744e6324a61b86b353d7b7b1ddc

**Comment by pandaring2you** [comment]
Current status: 
- Having issues with 0chain (ZCN) and Knit Finance (KFT) - James is investigating, might not be able to list tokens
- FTX Token will not be able to be listed until Peggy 2.0

Once #1854 is merged into develop, Aidan will push the list live. Will then need someone on front end to display tokens in the UI.
